.github/modules/retry/action.yml

```diff
@@ -0,0 +1,26 @@
+name: "Retry Steps"
+inputs:
+  command:
+    required: true
+  working-directory:
+    required: false
+    default: "./"
+  env-vars:
+    required: false
+    default: ""
+runs:
+  using: "composite"
```

Comment:
Add some documentation why we are using `composite` in this custom GHA module please.

Reply:
added at top of file